eve/bin/os_script_init.sh

```diff
@@ -54,9 +52,15 @@ then
     # buildbot > 0.9.12 requires a version of twisted that is not supported
     # by python2.6
     worker_version=0.9.12
+    python_version=2.6
   else
     twisted_version=16.4.0
+    python_version=2.7
   fi
+
+  curl https://bootstrap.pypa.io/pip/${python_version}/get-pip.py -o get-pip.py
```

Comment:
What is the reason to not use 'pip install --reinstall pip==9.0.3' ?
Do we really want to rely on another externally hosted script for thoat ?
Thats worth a comment at least to me